src/parser/rogue/outlaw/modules/spells/BetweenTheEyes.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatNumber, formatPercentage } from 'common/format';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+
+import { t } from '@lingui/macro';
+
+import { NumberThreshold, ThresholdStyle, When } from 'parser/core/ParseResults';
+
+import Events, { EventType, UpdateSpellUsableEvent } from 'parser/core/Events';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+
+import AlwaysBeCasting from '../features/AlwaysBeCasting';
+
+class BetweenTheEyes extends Analyzer {
+
+  timestampFromCD: number = 0;
+  totalTimeOffCD: number = 0;
+  isFirstCast: boolean = true;
+
+  static dependencies = {
+    spellUsable: SpellUsable,
+    alwaysBeCasting: AlwaysBeCasting
+  };
+  protected spellUsable!: SpellUsable;
+  protected alwaysBeCasting!: AlwaysBeCasting;
+
+  constructor(options: Options) {
+    super(options);
+    this.addEventListener(Events.UpdateSpellUsable.by(SELECTED_PLAYER).spell(SPELLS.BETWEEN_THE_EYES), this.onBetweenTheEyesUsable);
+  }
+
+  onBetweenTheEyesUsable(event: UpdateSpellUsableEvent) {
+    switch (event.trigger) {
+
+      case EventType.BeginCooldown: {
+
+        if (!this.isFirstCast) { 
+          this.totalTimeOffCD += event.timestamp - this.timestampFromCD;
+        } else {
+          this.isFirstCast = false;
+        }
+        break;
+      }
+        
+      case EventType.EndCooldown: {
+        this.timestampFromCD = event.timestamp;
+        break;
+      }
+    }
+  }
+
+  // Thresholds get retrieved at the end of analyzing
+  get thresholds(): NumberThreshold {
+    return {
+      actual: this.percentTimeOnCD,
+      isLessThan: {
+        minor: 0.9,
+        average: 0.80,
+        major: 0.75,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  get secondsOffCD(): number {
+    return (this.totalTimeOffCD / 1000);
+  }
+
+  get percentTimeOffCD(): number {
+    return this.secondsOffCD / (this.owner.fightDuration / 1000);
+  }
+
+  get percentTimeOnCD(): number {
+    return 1 - this.percentTimeOffCD;
+  }
+
+  get inefficientCastSuggestion(): JSX.Element {
```

Comment:
Return type is unnecessary and TypeScript gets easier to work with if you let it figure it our for itself, but you're free to keeping doing so if you prefer.